src/Nethermind/Nethermind.Merge.Plugin/IEngineRpcModule.Prague.cs

```diff
@@ -17,23 +18,11 @@ public partial interface IEngineRpcModule : IRpcModule
         Description = "Verifies the payload according to the execution environment rules and returns the verification status and hash of the last valid block.",
         IsSharable = true,
         IsImplemented = true)]
-    Task<ResultWrapper<PayloadStatusV1>> engine_newPayloadV4(ExecutionPayloadV4 executionPayload, byte[]?[] blobVersionedHashes, Hash256? parentBeaconBlockRoot);
+    Task<ResultWrapper<PayloadStatusV1>> engine_newPayloadV4(ExecutionPayloadV3 executionPayload, byte[]?[] blobVersionedHashes, Hash256? parentBeaconBlockRoot, byte[][]? executionRequests);
```

Comment:
Is it possible for `executionRequests` to be null in prague?

Reply:
No, it can't. I verify it here : https://github.com/NethermindEth/nethermind/blob/24673f2bf44fc4ae80c36ace1b569d5193bdb0c6/src/Nethermind/Nethermind.Merge.Plugin/Data/IExecutionPayloadParams.cs#L47